include/aws/s3/private/s3_parallel_read_stream.h

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#ifndef AWS_S3_PARALLEL_READ_STREAM_H
+#define AWS_S3_PARALLEL_READ_STREAM_H
+
+#include <aws/s3/s3.h>
+
+#include <aws/common/ref_count.h>
+
+AWS_PUSH_SANE_WARNING_LEVEL
+
+struct aws_byte_buf;
+struct aws_future_bool;
+struct aws_input_stream;
+
+struct aws_event_loop_group;
+
+struct aws_parallel_input_stream {
```

Comment:
trivial: struct is named "parallel_input_stream" but file is named "parallel_read_stream"

name the file: s3_parallel_stream.h/c or s3_parallel_input_stream.h/c